### DISCUSSION TITLE
Data source proxy call doesn't work

### DISCUSSION THREAD

**Discussion by evelinabut** [grafana/grafana#51690] [Data Sources]
Hi 

I am trying to query a MySQL Datasource via api:
 /api/datasources/proxy/:datasourceId/*

Example in the browser:
https://my-grafana-url/api/datasources/proxy/id/query?db=db_name&q=SELECT%20*%20from%20table_name

but nothing is returned back (just a blank page). 

In the log there is this error:

logger=data-proxy-log .... msg="Proxy request failed" err="net/http: HTTP/1.x transport connection broken: malformed HTTP response \"Y\\x00\\x00\\x00\""

What could be the problem? on the proxy endpoint?

**Comment by gabor** [comment]
hi @evelinabut i do not think this approach is going to work, the queried database ( `mysql` ) does not speak HTTP,  which i think is why the error message complains about a malformed HTTP response.

also please note that the mysql datasoure is a backend datasource plugin, so when it is running in the browser, it does not use the datasource proxy internally.

i hope this explains the situation. 

if you think there is a bug there and something here should work, please consider opening a bug report issue. thanks!
